students/astrawmyer/lesson04/mailroom.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# new file for lesson 4 work
+
+#donors = [["Manny Machado",[12.2,2.51,3.20]],["Adam Jones",[1024.14,22.21,323.45]],["Chris Davis",[3.2,5.55,4.20]]]
+ddonors = {"Manny Machado": [12.2,2.51,3.20], "Adam Jones": [1024.14,22.21,323.45], "Chris Davis": [3.2,5.55,4.20]}
+
+
+def displaylist():    
+    for name in ddonors.keys():
+        print(name)
+
+def writeletter(name,amount):
+    line_one = 'Dear {},'.format(name)
+    line_two = "Thank you for donating ${:.2f} to the Human Fund. Your money will be used appropriately.".format(amount)
+    letter = line_one + "\n" + line_two
+    return letter
+
+def writereport(donors):
+    print('Donor Name                | Total Given | Num Gifts | Average Gift')
+    print('-'*67)
+    for i in donors:
+        print('{1:27}${0:11.2f}{2:12}  ${3:12.2f}'.format(*i))
+
+
+# Function to send a thank you letter.
+
+def thankyou():
+    loop_trigger = True
+    while loop_trigger == True:
```

Comment:
True should use `is` comparison, this also applies to `is None` (for object comparison)